Composer/packages/extensions/adaptive-form/src/components/fields/ObjectArrayField.tsx

```diff
@@ -20,6 +20,12 @@ import { objectArrayField } from './styles';
 import { ArrayFieldItem } from './ArrayFieldItem';
 import { UnsupportedField } from './UnsupportedField';
 
+const END_OF_ROW_LABEL = formatMessage('press Enter to add this item or Tab to move to the next interactive element');
+
+const INSIDE_ROW_LABEL = formatMessage(
+  'press Enter to add this name and advance to the next row, or press Tab to advance to the value field'
+);
```

Comment:
These need to be defined inside the function or wrapped in functions to be invoked in the component. `formatMessage` needs to be able to translate just-in-time. This should produce a warning in the console.